### PR TITLE
#31: Popup not appearing on 13.x if the XWiki panels have been disabled

### PR DESCRIPTION
**Pull request by slauriere**
* Load effects.js via RequireJS in Core JSX
* Define Core JSX as 'cookie-consent' and add dependency on it in the
  Initializer JSX

### CODE REVIEW

**Comment by slauriere** [comment]
Hi @ppantiru, Hi all, This PR is an attempt to fix #31, which works fine on my side without any panel. Happy to know if you think it can be integrated or if any update is needed. /Cheers

**Comment by aubincleme** [comment]
Added a new check to verify if the scriptaculous effects package is already defined as a webjar. 

@mflorea anything from you that could block a merge ?

File: src/main/resources/CookieConsent/Script.xml [hunk 1]
```diff
@@ -122,147 +122,157 @@
...
-    if(!this.getCookie('cookieConsent')){
-      this.showDisclaimer();
-    }
-    var resetCookiesButton = $('ResetCookies');
-    if (resetCookiesButton != null) {
-      resetCookiesButton.observe('click', function() {
+      <code>require.config({
+  paths: {
+    'effect': "$xwiki.getSkinFile('js/scriptaculous/effects.js', true)" +
+      "?v=$escapetool.url($xwiki.version)"
```
[the remark this answers is not in the document]
Comment (slauriere) [reply src/main/resources/CookieConsent/Script.xml]: Thanks for the review @mflorea and for the recommendation. Sorry for not replying earlier, I just noticed your message today as my GitHub mail notification configuration was broken, I just fixed that.

**Comment by aubincleme** [comment]
Considering that this is a blocker for one of our clients ; I'll do the merge now so that we can issue a patch version that works on 13.x. We can still do updates afterwards.

**Pull request merged**